chess_env.py

```diff
@@ -16,6 +17,9 @@ def reset(self, state=None):
         :param state:
         :return:
         """
+        if state == None:
+            self.reset()
+
```

Comment:
we should be able to set the state to any position we want. That's what the state=None argument in line 14 is for

Reply:
also, make sure you make a new branch from develop when starting a new task. This should just be in your other pull request